src/app/contracts/lottery/Lottery.aes

```diff
@@ -0,0 +1,82 @@
+include "List.aes"
+contract Lottery=
```

Comment:
Since we are presenting this to the public, it would be good to give the code elegant look. Thus, I suggest adding spaces around all `=` signs, so `Lottery =` instead of `Lottery=`. This applies to type operator `:` and case-trigger (`=>`). Also, spaces after `,` in function args, so `Chain.spend(contestant'.user, valueToSend')` instead of `Chain.spend(contestant'.user,valueToSend')`